docs/nlu/using-bert.rst

```diff
@@ -0,0 +1,159 @@
+    - name: CountVectorsFeaturizer
+    - name: CountVectorsFeaturizer
+    analyzer: char_wb
+    min_ngram: 1
+    max_ngram: 4
+    - name: DIETClassifier
+    epochs: 20
+
+config/config-heavy.yml 
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: yaml
+
+    language: en
+    pipeline:
+    - name: HFTransformersNLP
+    model_weights: "bert-base-uncased"
+    model_name: "bert"
+    - name: LanguageModelTokenizer
+    - name: LanguageModelFeaturizer
+    - name: DIETClassifier
+    epochs: 30
+
+Note the differences in these files. 
+
+In the light configuration we have :ref:`CountVectorsFeaturizer` which create bag-of-word
+representations for each incoming message(at word and character levels). The heavy configuration replaces it with a
+BERT model inside the pipeline. :ref:`HFTransformersNLP` is a utility component that does the heavy lifting work of loading the
+``BERT`` model in memory. Under the hood it leverages HuggingFace's `Transformers library <https://huggingface.co/transformers/>`_ to initialize the specified language model.
+Notice that we add two additional components :ref:`LanguageModelTokenizer` and :ref:`LanguageModelFeaturizer` which
+pick up the tokens and feature vectors respectively that are constructed by the utility component.
+
+We use the same :ref:`diet-classifier` model for combined intent classification and entity recognition in both cases.
+
+Run the Pipelines
+-----------------
+
+You can run both configurations yourself.
+
+.. code-block:: yaml
+
+    mkdir gridresults
+    rasa test nlu --config configs/config-light.yml \
+                  --cross-validation --runs 1 --folds 2 \
+                  --out gridresults/config-light
+    rasa test nlu --config configs/config-heavy.yml \
+                  --cross-validation --runs 1 --folds 2 \
+                  --out gridresults/config-heavy
+
+When this runs you should see logs appear. We've picked a few
+of those lines to list them here. 
+
+.. code-block:: txt
+
+    # output from the light model
+    2020-03-30 16:21:54 INFO     rasa.nlu.model  - Starting to train component DIETClassifier
+    Epochs: 100%|███████████████████████████████| 50/50 [04:30<00:00, ...]
+    2020-03-30 16:23:53 INFO     rasa.nlu.test  - Running model for predictions:
+    100%|███████████████████████████████████████| 2396/2396 [01:23<00:00, 28.65it/s]
+    ...
+    # output from the heavy model
+    2020-03-30 16:47:04 INFO     rasa.nlu.model  - Starting to train component DIETClassifier
+    Epochs: 100%|███████████████████████████████| 50/50 [04:33<00:00,  ...]
+    2020-03-30 16:49:52 INFO     rasa.nlu.test  - Running model for predictions:
+    100%|███████████████████████████████████████| 2396/2396 [07:20<00:00,  5.69it/s]
+
+From the logs we can gather an important observation. 
+The heavy model is a fair bit slower, not in training, but at inference time
+we see a ~6 fold increase. Depending on your use-case this is 
+something to seriously consider.
```

Comment:
This is an important observation. Can we highlight this in some manner. For some reason, I just skipped this observation in my first reading. Two options - 
1. Move it to the observations section below, so that all observations are in one place and reader doesn't miss them
2. Or create a `note` block and put this in that so that it is highlighted.
What do you think?

Reply:
I've added a note and moved it just before the results.